integ-tests/teradata/src/main/java/com/google/tdjdbc/JdbcUtil.java

```diff
@@ -91,7 +92,11 @@ public static double getDoubleNotNull(ResultSet rs, String column) throws SQLExc
    * @return long or 0L if null.
    */
   public static long getTimestampNotNull(ResultSet rs, String column) throws SQLException {
-    Timestamp timestamp = rs.getTimestamp(column);
-    return rs.wasNull() ? 0L : timestamp.getTime();
+    Calendar cal = Calendar.getInstance();
+    Timestamp timestamp = rs.getTimestamp(column, cal);
+    if (rs.wasNull()) return 0L;
```

Comment:
Can you run `git fix` please to ensure the formatting is according to the style guide? This line should violate https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

Reply:
I did run google-java formatter repeatedly and strangely it doesn't correct this according to the javaguide. Should we report this bug?